### PR TITLE
muti-replication 1.2

### PR DESCRIPTION
**Pull request by PhMaly**
Add docker compose, init.sql, scritp dump, readme.

### CODE REVIEW

File: Philemon/multi-replication/dump-et-sauvegarde/dump.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
```
Comment (fauust) [inline Philemon/multi-replication/dump-et-sauvegarde/dump.sh:2]:
```bash
set -e
```

File: Philemon/multi-replication/dump-et-sauvegarde/dump.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+timestamp=$(date +%F_%T)
+
+# Check if sql-01 container is running
+if docker ps --format '{{.Names}}' | grep -q 'sql-01'; then
+  # backup db
+  docker exec sql-01 bash -c "mariadb-dump -v --all-databases -u root -ptest123 > /shared-data/backup-db-$timestamp.sql"
+  # appear in the logs journalctl -f
+  logger "Backup completed for sql-01 at $timestamp."
+else
+  logger "Container sql-01 is not running. Skipping backup."
```
Comment (fauust) [inline Philemon/multi-replication/dump-et-sauvegarde/dump.sh:12]: exit 1

**Review by fauust — CHANGES_REQUESTED** [review]

File: Philemon/multi-replication/dump-et-sauvegarde/dump.sh (hunk 1 above)
Comment (PhMaly) [reply Philemon/multi-replication/dump-et-sauvegarde/dump.sh:2]: ok

File: Philemon/multi-replication/dump-et-sauvegarde/dump.sh (hunk 2 above)
Comment (PhMaly) [reply Philemon/multi-replication/dump-et-sauvegarde/dump.sh:14]: ok

**Pull request merged**